source/reference/parameters.txt

```diff
@@ -42,6 +42,8 @@ For additional configuration options, see
 :doc:`/reference/configuration-options`, :binary:`~bin.mongod` and
 :binary:`~bin.mongos`.
 
+Changes made via the :dbcommand:`setParameter` will revert to defaults after reboot unless they're included in either the boot time configuration file or the ``--setParameter`` command-line option.
```

Comment:
I think this note should be under the section that deals with setParameter call for further clarity.